vividus-plugin-excel/src/main/java/org/vividus/excel/ExcelSheetParser.java

```diff
@@ -225,6 +227,32 @@ public String getDataFromCell(String address)
                 .orElse(null);
     }
 
+    public Map<String, List<String>> exactDataTable(String range)
+    {
+        CellRangeAddress address = CellRangeAddress.valueOf(range);
+        Map<String, List<String>> data = new HashMap<>();
+        List<List<String>> listRow = new ArrayList<>();
+        for (int rowIndex = address.getFirstRow(); rowIndex <= address.getLastRow(); rowIndex++)
+        {
+            List<String> col = new ArrayList<>();
+            Row row = sheet.getRow(rowIndex);
+            for (int colIndex = address.getFirstColumn(); colIndex <= address.getLastColumn(); colIndex++)
+            {
+                col.add(row.getCell(colIndex).getStringCellValue());
+            }
+            listRow.add(col);
+        }
+        List<String> headers = listRow.get(0);
+        listRow.remove(0);
+        for (int i = 0; i < headers.size(); i++)
+        {
+            final int index = i;
+            List<String> listColumn = listRow.stream().map(list -> list.get(index)).collect(Collectors.toList());
```

Comment:
what about building table at excel traversal (without this extra transformation)?